packages/common-web/src/async.ts

```diff
@@ -1,17 +1,32 @@
-import { bailableWait } from './util'
+import { wait } from './util'
+
+export const emptyPromise: Promise<void> = new Promise((resolve) => resolve())
```

Comment:
No biggie here either way, just a more concise way to write the exact same thing:
```suggestion
export const emptyPromise: Promise<void> = Promise.resolve()
```

Reply:
oh far out didn't realize that